pkg/flare/archive.go

```diff
@@ -326,7 +326,32 @@ func zipExpVar(tempDir, hostname string) error {
 		}
 	}
 
-	return nil
+	apmPort := "8126"
+	if config.Datadog.IsSet("apm_config.receiver_port") {
+		apmPort = config.Datadog.GetString("apm_config.receiver_port")
+	}
+	resp, err := http.Get(fmt.Sprintf("http://127.0.0.1:%s/debug/vars", apmPort))
+	if err != nil || resp.StatusCode != http.StatusOK {
```

Comment:
When will `err != nil` but `StatusCode` be `StatusOK` ?

Reply:
It's an _OR_ condition. If err is not nil, it executes _OR_ if status is not OK 😄 You misread.

Reply:
:man_facepalming: